apps/src/code-studio/DisabledBubblesAlert.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react';
+import Alert from '@cdo/apps/templates/alert';
+import i18n from '@cdo/locale';
+
+const styles = {
+  bold: {
+    fontFamily: '"Gotham 5r", sans-serif'
+  }
+};
+
+/**
+ * Component that displays a small notification at the top of the page when
+ * postMilestone is disabled (and thus progress is disabled)
+ */
+const DisabledBubblesAlert = React.createClass({
+  propTypes: {
+    isHocScript: PropTypes.bool.isRequired
+  },
+
+  getInitialState() {
+    // Once alert has been dismissed, don't show again.
+    const disabledBubblesAlertSeen = sessionStorage.getItem('disabledBubblesAlertSeen');
```

Comment:
[From MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage):

> ...data stored in sessionStorage gets cleared when the page session ends. A page session lasts for as long as the browser is open and survives over page reloads and restores. Opening a page in a new tab or window will cause a new session to be initiated, which differs from how session cookies work.

This makes it sound like each _different_ page is its own session - so will this alert show up for every _different_ puzzle in an HOC progression?  And/or is that the intended behavior?  I got the impression we want this to show once per browser session, not per page session.